### PR TITLE
Add blockstore

### PR DESCRIPTION
**Pull request by mrtmm**
Add the blockstore role in hastexo-multi-node.yml

### CODE REVIEW

File: playbooks/hastexo-multi-node.yml [hunk 1]
```diff
@@ -46,6 +46,7 @@
       when: POSTFIX_QUEUE_EXTERNAL_SMTP_HOST != ''
     - guacd
     - hastexo_xblock
+    - blockstore
```
Comment (fghaas) [inline playbooks/hastexo-multi-node.yml:49]: We'll want this in `hastexo-single-node.yml` too.

And I'm not sure if the LMS only ever talks to blockstore locally, or whether it also needs to be exposed though nginx. If it's the latter, I'd assume we'd need a new item in the `nginx_sites` list too?

Comment (mrtmm) [reply playbooks/hastexo-multi-node.yml:50]: I'll go ahead and trust your assumption on that, hope that I got it right.

File: playbooks/hastexo-multi-node.yml [hunk 2]
```diff
@@ -35,6 +35,7 @@
       nginx_sites:
       - cms
       - lms
+      - blockstore
```
Comment (fghaas) [inline playbooks/hastexo-multi-node.yml:38]: I just checked the role and it apparently inherits its functionality from `edx-django-service`, and its service name is `blockstore`, so yes this should be good.

Comment (fghaas) [reply playbooks/hastexo-multi-node.yml:38]: Evidently not. :(

```
failed: [appmaster0] (item=blockstore) => {"failed": true, "item": "blockstore", "msg": "Unable to find 'edx/app/nginx/sites-available/blockstore.j2' in expected paths."}
```

Comment (mrtmm) [reply playbooks/hastexo-multi-node.yml:38]: Right. :/

Comment (mrtmm) [reply playbooks/hastexo-multi-node.yml:38]: I amended this commit with '-blockstore' moved to nginx_default_sites, that is tested now and works.

Comment (fghaas) [reply playbooks/hastexo-multi-node.yml:38]: Sounds good, thanks!

**Pull request merged**